src/registrar/admin.py

```diff
@@ -222,11 +256,97 @@ class ContactAdmin(ListHeaderAdmin):
 
     search_fields = ["email", "first_name", "last_name"]
     search_help_text = "Search by firstname, lastname or email."
+    list_display = [
+        "contact",
+        "email",
+    ]
+
+    # We name the custom prop 'contact' because linter
+    # is not allowing a short_description attr on it
+    # This gets around the linter limitation, for now.
+    def contact(self, obj):
+        if obj.first_name or obj.last_name:
```

Comment:
Truthy/Falsy values are fine to use but its generally recommended that you explicitly define the type you want to handle (for example, you may want to add a type on obj). These shorthands will return false in the following conditions:
![image](https://github.com/cisagov/getgov/assets/141044360/6fdd10ed-563f-4bfe-9aff-ce19d4d60836)
Basically, if for some reason a string gets passed in as a number, python may still inevitably parse that as a string once its rendered but you could run into buggy behavior. For instance, if obj.first_name was parsed as an integer of '0', then this field would never execute as while the first name is technically "0", python may assume otherwise

Reply:
obj is an instance of Contact, and I'm comparing against its props (first_name and last_name are models.TextField, etc.) Don't all these types trickle down into obj in the admin class when you link admin.site.register(models.Contact, ContactAdmin)?

Reply:
@rachidatecs That is correct, I think I misunderstood the purpose of the function. I'd assume your using this as a workaround as it doesn't allow you to call and/or invoke .__str__ directly. My original assumption was moreso that this could be used by a developer and they (may) pass in the wrong type but given the function is literally called 'contact' I think that would be implied (or may try to use it on an epp Contact object, for instance)... 

With that context in mind, what your doing here is fine. A nice to have might be to add a type hint on obj (obj: Contact) if it doesn't break anything just as a precaution, but really not necessary especially in this context.